src/index.css

```diff
@@ -1,5 +1,12 @@
 body {
-  margin: 0 auto;
   box-sizing: border-box;
   font-family: 'Courier New', Courier, monospace;
 }
+
+.box-container {
```

Comment:
- [ ] As the previous reviewer suggested, kindly stick to wireframe. Your app is not fitting to the webpage. Kindly fix it. 

This is the screenshot of your app:

<img width="925" alt="Screenshot 2023-03-16 at 09 36 17" src="https://user-images.githubusercontent.com/69011963/225515919-7d2ed64d-e224-4b53-86cb-a4cc58418587.png">

> HINT: I suggest using overflow scroll for displaying scores.